### PR TITLE
Add crd application tutorial

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add tutorial about how to propagate crd applications.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
None

### CODE REVIEW

File: docs/tutorials/crd-application.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+The CRD will be propagated to member clusters according to the rules defined in ClusterPropagationPolicy.
+
+> Note: We can only use ClusterPropagationPolicy not PropagationPolicy here.
+> Please refer to FAQ Difference between [PropagationPolicy and ClusterPropagationPolicy](https://github.com/karmada-io/karmada/blob/master/docs/frequently-asked-questions.md#what-is-the-difference-between-propagationpolicy-and-clusterpropagationpolicy)
```
Comment (RainbowMango) [inline docs/tutorials/crd-application.md:71]: The link is broken.

File: docs/tutorials/crd-application.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
+---
+title: Propagate a CRD application by Karmada
```
Comment (Tingtal) [inline docs/tutorials/crd-application.md:2]:
```suggestion
title: Propagate a CRD application with Karmada
```

File: docs/tutorials/crd-application.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+---
+This guide will cover:
```
Comment (Tingtal) [inline docs/tutorials/crd-application.md:4]:
```suggestion
This section guides you to:
```

File: docs/tutorials/crd-application.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+```sh
+git clone https://github.com/karmada-io/karmada
+cd karmada
+hack/local-up-karmada.sh
+```
+
+## Propagate CRD application
+
+The following steps demonstrating how to propagate a [Guestbook](https://book.kubebuilder.io/quick-start.html#create-a-project) which is defined by CRD.
```
Comment (Tingtal) [inline docs/tutorials/crd-application.md:23]:
```suggestion
The following steps walk you through how to propagate a [Guestbook](https://book.kubebuilder.io/quick-start.html#create-a-project) defined by a CRD.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/crd-application.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
+---
+title: Propagate a CRD application by Karmada
```
Comment (Tingtal) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/crd-application.md:2]: Sync the changes above. Thanks

**Review by Tingtal — CHANGES_REQUESTED** [review]
Several touches on wording

**Comment by RainbowMango** [comment]
Please solve the conflicts.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**